CHANGES.md

```diff
@@ -1,5 +1,25 @@
 # Changes
 
+## edge-20.7.1
+
+This edge release improves messaging for retries when pods are unschedulable,
+adds new functionality to persist prometheus data and configure the proxy log
+format using annotations and helm charts, fixes a bug in the helm chart for
+the Linkerd CNI Plugin, and enables the use of URLs with the `--addon-config`
+installation flag
+
+* Controller
+  * Adds better messaging during scheduling errors on retry for HA installations
+* CLI
+  * Adds option to persist prometheus data (thanks @naseemkullah!)
+  * Adds support for passing a URL to the `--addon-config` flag, so you can now
+  use a file or an URL for installing add ons
+* Helm
+  * Fixes a bug in the CNI helm chart by correcting a helm value field name
+* Proxy
+  * Improves tail latencies by increasing the default buffer size to reduce
+    contention in high-concurrency situations
```

Comment:
```suggestion
* Some commands like `linkerd stat` would fail would fail if any control plane components were unhealthy, even when other replicas are healthy. This checks have been relaxed.
* The helm chart can now configure persistent storage for Prometheus (thanks @naseemkullah!)
* `linkerd install --addon-config=` now supports URLs in addition to local files
* The CNI Helm chart used the incorrect variable to determine the container version. This is now controlled by `cniPluginVersion`.
* The proxy's default buffer sized has been increased, which reduces latency when the proxy has many concurrent clients.
```